.circleci/config.yml

```diff
@@ -82,6 +82,25 @@ jobs:
 
       - run: npm run test-cov
 
+  gh_pages:
+    executor: node
+
+    steps:
+      - attach_workspace:
+          at: .
+
+      - run:
+          name: Set Git credentials
+          command: |
+            echo 'machine github.com' >> ~/.netrc
+            echo "        login $GIT_AUTHOR_NAME" >> ~/.netrc
+            echo "        password $GH_TOKEN" >> ~/.netrc
+
+      - run:
+          name: Publish GitHub Pages
+          command: |
+            npx gh-pages --dist . --message "chore: update to $(git rev-parse HEAD) [ci skip]" --repo "$(node -e 'process.stdout.write(require("./package.json").repository.url)')" --src '{{coverage,dist,docs,examples}/**/*,LICENSE*,*.{html,md}}'
```

Comment:
We are currently not using the gh-pages branch but use master for the github-pages feature. This ways ew do not have wo sync two branches and can always be sure the examples are using the latest master version (e.g. useful for bug-reports).

I really would like to keep it that way!

Reply:
As long as everybody always updates everything manually, sure. But what if someone doesn't rebuild dist in PR? That can happen and already happened. This way everything would happen automatically in CI.

Reply:
Sounds reasonable!
I'll try to have a look this week and try to figure out if this breaks something...